docs/share_usage.md

```diff
@@ -35,46 +42,60 @@ suppress_requests_ssl_warnings = True
 # Set the driver to be File driver
 hpedockerplugin_driver = hpedockerplugin.hpe.hpe_3par_file.HPE3PARFileDriver
 
-hpe3par_api_url = https://10.50.3.24:8080/api/v1
-hpe3par_username = 3paradm
-hpe3par_password = 3pardata
-hpe3par_san_ip = 10.50.3.24
-hpe3par_san_login = 3paradm
-hpe3par_san_password = 3pardata
+hpe3par_api_url = https://xxx.xxx.xxx.xxx:8080/api/v1
+hpe3par_username = <user>
+hpe3par_password = <pwd>
+hpe3par_san_ip = xxx.xxx.xxx.xxx
+hpe3par_san_login = <san_user>
+hpe3par_san_password = <san_pwd>
 
 # Server IP pool is mandatory and can be specified as a mix of range of IPs and
 # individual IPs delimited by comma
 # Each range or individual IP must be followed by the corresponding subnet mask
 # delimited by semi-colon
 # E.g.: IP-Range:Subnet-Mask,Individual-IP:SubnetMask…
```

Comment:
One more line needs to be added.
It is mandatory that IPs provided here are reachable from host after creation of vfs.

Reply:
I see a "Note" section providing this information. Hence above comment can be ignored